config/api.yml

```diff
@@ -461,6 +461,9 @@
         :identifier: miq_report_run
     :subcollections:
     - :results
+    :collection_actions:
+      :post:
+      - :name: import
```

Comment:
I think "miq_report_export" may be fine since it's product feature name is "Import / Export".

Dan would know better than I,  @dclarizio thoughts ?